drivers/bus/acpi_new/CMakeLists.txt

```
@@ -0,0 +1,52 @@
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
+
+include_directories(
+    uacpi/include)
+include_directories(
+    "uacpi/include/uacpi/")
+include_directories(
+    "uacpi/include/uacpi/platform")
+include_directories(
+    "uacpi/include/uacpi/internal")
```

```suggestion
include_directories(
    uacpi/include
    "uacpi/include/uacpi/"
    "uacpi/include/uacpi/platform"
    "uacpi/include/uacpi/internal")
```
Note also, that we don't usually put the quotes when the paths don't contain spaces.